packages/cli/src/commands/setup/shadowenv/shadowenv.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+
+const shadowenvConfigPath = `${getPaths().base}/.shadowenv.d/rw.lisp`
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr(
+    [
+      {
+        title: 'Configuring Shadowenv...',
+        task: (_, task) => {
+          /**
+           * Check if Shadowenv config already exists.
+           * If it exists, throw an error.
+           */
+          if (!force && fs.existsSync(shadowenvConfigPath)) {
+            throw new Error(
+              'Shadowenv config already exists.\nUse --force to override existing config.'
+            )
+          } else {
+            return writeFile(
+              shadowenvConfigPath,
+              fs
+                .readFileSync(
+                  path.resolve(__dirname, 'templates', 'rw.lisp.template')
+                )
+                .toString(),
+              {
+                overwriteExisting: force,
+              },
+              task
+            )
+          }
+        },
+      },
+      {
```

Comment:
Ah, I see we need to add a recommended step, which is gitignore‘ing the entire .shadowenv.d directory. See: https://shopify.github.io/shadowenv/best-practices/#gitignore

You can effectively duplicate the step `title: 'Updating .gitignore...',` from the deploy.js command here:
https://github.com/redwoodjs/redwood/blob/bf9530fc7fe5b97f77e4a3656c9c0f4f88510cee/packages/cli/src/commands/setup/deploy/deploy.js#L169-L191